scripts/create_quantile_table.sql

```diff
@@ -1,9 +1,9 @@
 CREATE TABLE PUMS_2014_Quantiles (
-  "QUANTILE" VARCHAR(10),
-  "PUMA" VARCHAR(5),
-  "STATE" VARCHAR(3),
-  "AGEGROUP" VARCHAR(20),
-  "RACE" VARCHAR(20),
-  "SEX" VARCHAR(10),
-  "INCOME" FLOAT8
+  QUANTILE VARCHAR(10),
+  PUMA VARCHAR(5),
+  STATE VARCHAR(3),
+  AGEGROUP VARCHAR(20),
+  RACE VARCHAR(20),
+  SEX VARCHAR(10),
+  INCOME FLOAT8
```

Comment:
Generally it's best to avoid `FLOAT`s and go for `NUMERIC` or `DECIMAL` types. I wonder if the imprecision of the float type could have affected some minor differences in the output numbers, like you were seeing. We can leave this as-is for now, but we might want to look into changing this in the future.